src/MicrosoftTeams.ts

```diff
@@ -48,12 +48,14 @@ namespace microsoftTeams
     // It does not indicate whether initialization is complete. That can be inferred by whether parentOrigin is set.
     let initializeCalled = false;
 
+    let currentWindow: Window;
     let parentWindow: Window;
     let parentOrigin: string;
     let messageQueue: MessageRequest[] = [];
     let nextMessageId = 0;
     let callbacks: {[id: number]: Function} = {};
     let frameContext: string;
+    let loadedInDesktopClient: boolean;
```

Comment:
Yeah, I will change it to an enum on my next iteration. Right now it will just have Web and Desktop but we can add more values to it later.